src/sentry/models/auditlogentry.py

```diff
@@ -369,11 +387,14 @@ def get_note(self):
         elif self.event == AuditLogEntryEvent.SET_ONDEMAND:
             if self.data["ondemand"] == -1:
                 return "changed on-demand spend to unlimited"
-            next_ondemand_max_spend = format_ondemand_max_spend(self.data["ondemand"])
+            next_ondemand_budget = format_ondemand_budget(self.data["ondemand"])
             if "prev_ondemand" in self.data:
-                prev_ondemand_max_spend = format_ondemand_max_spend(self.data["prev_ondemand"])
-                return f"changed on-demand max spend from {prev_ondemand_max_spend} to {next_ondemand_max_spend}"
-            return f"changed on-demand max spend to {next_ondemand_max_spend}"
+                ondemand_label = "on-demand max spend"
+                if type(self.data["ondemand"]) is dict or type(self.data["prev_ondemand"]) is dict:
+                    ondemand_label = "on-demand budget"
```

Comment:
Couldn't we always use the term `on-demand budget`? It is a suitable term for both on-demand implementations.